inc/ticket.class.php

```diff
@@ -6707,12 +6765,13 @@ function getTimelineItems() {
 
          $timeline[$solution_date."_solution"]
             = ['type' => 'Solution',
-                    'item' => ['id'               => 0,
-                                    'content'          => Toolbox::unclean_cross_side_scripting_deep($solution_content),
-                                    'date'             => $solution_date,
-                                    'users_id'         => $users_id,
-                                    'solutiontypes_id' => $this->fields['solutiontypes_id'],
-                                    'can_edit'         => Ticket::canUpdate() && $this->canSolve()]];
+                     'item' => ['id'                => 0,
+                                      'content'           => Toolbox::unclean_cross_side_scripting_deep($solution_content),
```

Comment:
CS: align the array lines.
(not your fault originally, but in the current workflow , if we edit a part like this, we realign the arrays)